content/hi/docs/concepts/architecture/cgroups.md

```diff
@@ -0,0 +1,127 @@
+
+### लिनक्स वितरण सीग्रुप v2 समर्थन
+
+
+सीग्रुप v2 का उपयोग करने वाले Linux वितरणों की सूची के लिए,  [सीग्रुप v2 दस्तावेज़](https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md) देखें
+
+<!-- the list should be kept in sync with https://github.com/opencontainers/runc/blob/main/docs/cgroup-v2.md -->
+* कंटेनर अनुकूलित ओएस (एम97 से)
+* उबंटू (21.10 से, 22.04+ अनुशंसित)
+* डेबियन जीएनयू/लिनक्स (डेबियन 11 बुल्सआई के बाद से)
+* फेडोरा (31 से)
+* आर्क लिनक्स (अप्रैल 2021 से)
+* आरएचईएल और आरएचईएल-जैसे वितरण (9 से)
+
+यह जाँचने के लिए कि क्या आपका वितरण cgroup v2 का उपयोग कर रहा है, अपने वितरण के दस्तावेज़ देखें या [Linux नोड्स पर cgroup संस्करण की पहचान](#check-cgroup-version) करें में दिए गए निर्देशों का पालन करें।
+
+आप कर्नेल cmdline बूट तर्कों को संशोधित करके अपने लिनक्स वितरण पर सीग्रुप v2 को मैन्युअल रूप से भी सक्षम कर सकते हैं। यदि आपका वितरण GRUB का उपयोग करता है, `systemd.unified_cgroup_hierarchy=1` को `GRUB_CMDLINE_LINUX` में जोड़ा जाना चाहिए
+`/etc/default/grub` के अंतर्गत, उसके बाद `sudo update-grub`। हालांकि
+अनुशंसित दृष्टिकोण एक ऐसे वितरण का उपयोग करना है जो पहले से ही सीग्रुप v2 को सक्षम बनाता है
+गलती करना।
+
+### सीग्रुप v2 पर माइग्रेट करें {#migrating-cgroupv2}
+
+cgroup v2 में प्रवास करने के लिए सुनिश्चित करें कि आप [आवश्यकताओं](#requirements) को पूरा करते हैं, और फिर उस कर्णल संस्करण पर अपग्रेड करें जिसमें cgroup v2 को डिफ़ॉल्ट रूप से सक्षम किया गया है।
+
+कुबलेट स्वचालित रूप से पहचानता है कि ऑपरेटिंग सिस्टम cgroup v2 पर चल रहा है और बिना किसी अतिरिक्त कॉन्फ़िगरेशन के यह उचित रूप से कार्रवाई करता है।
+
+cgroup v2 में स्विच करते समय उपयोगकर्ता अनुभव में कोई भी प्रमुख अंतर होना नहीं चाहिए, यहाँ तक कि यदि उपयोगकर्ता सीधे cgroup फ़ाइल सिस्टम तक पहुँच रहे हैं, या नोड से या कंटेनर के भीतर से।
+
+cgroup v2 cgroup v1 की तुलना में एक अलग API का उपयोग करता है, इसलिए यदि कोई एप्लिकेशन सीधे cgroup फ़ाइल सिस्टम तक सीधे पहुँचता है, तो उन्हें cgroup v2 को समर्थन करने वाले नए संस्करणों में अपडेट किया जाना चाहिए। उदाहरण के लिए:
+
+* Some third-party monitoring and security agents may depend on the cgroup filesystem.
+ Update these agents to versions that support cgroup v2.
+* If you run [cAdvisor](https://github.com/google/cadvisor) as a stand-alone
+ DaemonSet for monitoring pods and containers, update it to v0.43.0 or later.
+* If you deploy Java applications, prefer to use versions which fully support cgroup v2:
+    * [OpenJDK / HotSpot](https://bugs.openjdk.org/browse/JDK-8230305): jdk8u372, 11.0.16, 15 and later
+    * [IBM Semeru Runtimes](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.382.0, 11.0.20.0, 17.0.8.0, and later
+    * [IBM Java](https://www.ibm.com/support/pages/apar/IJ46681): 8.0.8.6 and later
+* If you are using the [uber-go/automaxprocs](https://github.com/uber-go/automaxprocs) package, make sure
+  the version you use is v1.5.1 or higher.
```

Comment:
@anshitmishra Thanks for you contribution. There are still some lines in English; it would be helpful to have them localized in Hindi as well.

Reply:
yaa sure working on it now